utils/freshmeat/Classes/FreshmeatRdfs.php

```diff
@@ -153,7 +153,7 @@ public function XtractProjInfo($rdf_file)
       );
       foreach ($project->latest_release as $verdata)
       {
-        array_push(& $this->project_info["$project->projectname_short"],
+        array_push($this->project_info["$project->projectname_short"],
         $verdata->latest_release_version
                    );
```

Comment:
Can be optimized for single insertion by using array operator as bellow:
```suggestion
        $this->project_info["$project->projectname_short"][] = $verdata->latest_release_version;
```